fastapi/openapi/models.py

```diff
@@ -24,13 +23,13 @@ class EmailStr(str):  # type: ignore
 
 class Contact(BaseModel):
     name: Optional[str] = None
-    url: Optional[UrlStr] = None
+    url: Optional[AnyUrl] = None
```

Comment:
There are a few new url types, but `AnyUrl` is the only one that is compatible with everything that `UrlStr` was compatible with before (namely, `ftp` and `ftps` schemes). `AnyHttpUrl` or `HttpUrl` would work if we are okay enforcing the scheme being `http` or `https`. (`AnyHttpUrl` allows there to not be a tld, `HttpUrl` requires it; it seems to me like `HttpUrl` would be fine here but the downside of using `AnyUrl` seems pretty minimal, and it should prevent breaking anything.)

Reply:
Perfect, thanks for the explanation. :rocket: